gradle-maven-push.gradle

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright 2013 Chris Banes
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'maven'
+apply plugin: 'signing'
+
+version = VERSION_NAME
```

Comment:
@rossbacher do you mean `PROJECT_VERSION`? Also you set `version` and `group`, but don't seem to use them

Reply:
@mmdango Yes, I created a merge conflict here. I just pushed a fix.

Reply:
@mmdango Does not look like it, I'm removing them.